src/test/java/uk/co/ramp/covid/simulation/CovidTest.java

```diff
@@ -56,18 +56,52 @@ public void testStepInfectionRecover() throws IOException {
         CStatus cStatus = null;
         Person child = new Child();
         Covid virus = new Covid(child);
+        virus.forceSymptomatic(true);
 
-        int latentPeriod = virus.getLatentPeriod() * 24;
-        double asymptomaticPeriod = virus.getAsymptomaticPeriod() * 24;
-        double p1Period = virus.getP1() * 24;
+        double latentPeriod = virus.getLatentPeriod();
+        double asymptomaticPeriod = virus.getAsymptomaticPeriod();
+        double p1Period = virus.getP1();
 
         //Test that the person becomes recovered after the total infection period
-        for (int i = 0; i < latentPeriod + asymptomaticPeriod + p1Period; i++) {
+        for (int i = 0; i < latentPeriod + p1Period; i++) {
             cStatus = virus.stepInfection();
         }
+      //  cStatus = virus.stepInfection();
         assertEquals(CStatus.RECOVERED, cStatus);
+        assertEquals(false, virus.getIsSymptomatic());
     }
 
+    //Test that a child steps through the infection from latent to recovered
+    @Test
+    public void testStepInfectionAsymptomatic() throws IOException {
+        RNG.seed(321);
+        ParameterReader.readParametersFromFile("src/test/resources/default_params.json");
+        Person child = new Child();
+        Covid virus = new Covid(child);
+        virus.forceSymptomatic(false);
+
+        double latentPeriod = virus.getLatentPeriod();
+        double asymptomaticPeriod = virus.getAsymptomaticPeriod();
+
+        CStatus cStatus = virus.stepInfection();
+        
+        //Test that the person becomes recovered after the total infection period
+        for (int i = 1; i < latentPeriod; i++) {
+            assertEquals(CStatus.LATENT, cStatus);
+            assertEquals(false, virus.getIsSymptomatic());
+        	cStatus = virus.stepInfection();
+        }
+        
+        for (int i = 0; i < asymptomaticPeriod; i++) {
+            assertEquals(CStatus.ASYMPTOMATIC, cStatus);
+            assertEquals(false, virus.getIsSymptomatic());
+        	cStatus = virus.stepInfection();
+        }
+//        cStatus = virus.stepInfection();
```

Comment:
Probably best to remove this so it doesn't confuse people whether it should be there or not!

Reply:
Done